pallets/ocex/src/lib.rs

```diff
@@ -366,6 +368,17 @@ pub mod pallet {
 				!<TradingPairs<T>>::contains_key(quote, base),
 				Error::<T>::TradingPairAlreadyRegistered
 			);
+			//enclave will only support min volume of 10^-8, if the trading pair register
+			//falls below min volume limits it will send out an error
+			ensure!(
+				(Decimal::from(min_order_price.saturated_into::<u128>())
+					.div(&Decimal::from(UNIT_BALANCE)))
```

Comment:
Decimal::from will panic if min_order_price is greater than Decimal::MAX.

Reply:
collect_fees, claim_withdraw, deposit, register_trading_pair all have Decimal::from should we convert it to Decimal::from_f64 and handle it same thing as we have done [here in OB](https://github.com/Polkadex-Substrate/orderbook/blob/72ff75be699eb56983ac488db45845e1e29439bc/primitives/src/orderbook.rs#L269)

Reply:
This is updated in #522 
I've added a check that ensures `Decimal::from` will not panic, as we return an error if any param is greater than `10^27`.

Reply:
Resolved checked for min value of 1000 u128